wiki/markdown/artifact-created-event-aggregation.md

```diff
@@ -138,7 +138,7 @@ Once the information extraction is complete, the next processing steps are
 different for start events (an event with no linked aggregated object) and
 events with link to an associated aggregated object. For start event, an object
 is created out of the extracted content and inserted into the database, with a
-name configured in the properties file as “aggregated.collection.name”. The
+name configured in the properties file as “aggregations.collection.name”. The
```

Comment:
Same comment here. In this particular context the reader is more interested in the flow and we could instead write some thing like: "\[The database collection name can be configured\]\(configuration.md\)"

Reply:
Done